Makefile

```diff
@@ -0,0 +1,32 @@
+VERSION?=0.0.0
+
+.PHONY: build
+build: build-py
+	@echo "Built $(VERSION)"
+
+.PHONY: build-py
+build-py:
+	@echo "Building Python package" && \
+	cp data/public.gpg python/insights_ansible_playbook_verifier/data/public.gpg
+	cp data/revoked_playbooks.yml python/insights_ansible_playbook_verifier/data/revoked_playbooks.yml
+	sed -i "s|version = .*|version = '$(VERSION)'|" pyproject.toml
```

Comment:
hmm rather than replacing the version here, what about using dynamic ways to read the version in pyproject.toml? from what I see here
https://packaging.python.org/en/latest/guides/single-sourcing-package-version/#single-sourcing-the-version
it seems to be possible; maybe the "base" version defined statically in a `VERSION` file, and an optional git suffix coming from an envvar?

Reply:
We would still need to write it down/update it in this target. How is a separate file different from updating package metadata?